lib/network_utils.ml

```diff
@@ -39,7 +39,15 @@ let inject_igmp_query_script = ref "/usr/libexec/xenopsd/igmp_query_injector.py"
 let mac_table_size = ref 10000
 let igmp_query_maxresp_time = ref "5000"
 
+let in_unit_test = ref false
+let unit_test_call_script = ref ""
+
 let call_script ?(log_successful_output=false) ?(timeout=Some 60.0) script args =
+	if !in_unit_test then begin
```

Comment:
This is all a bit odd. If we really want to do this, then we should functorise this bit as done for the Ovs module. But as I said in my other comment, I don't think this is really worth it at all.